pkgs/top-level/all-packages.nix

```diff
@@ -16090,7 +16090,7 @@ in
 
   newman = callPackage ../development/web/newman {};
 
-  newt = callPackage ../development/libraries/newt { };
+  newt = callPackage ../development/libraries/newt { python = python3; };
```

Comment:
Can we replace python in inputs with python3?

Reply:
not without modifying pythonPackages